index.js

```diff
@@ -38,8 +32,16 @@ module.exports = {
     "blank-lines": "off",
     "arg-overflow": "off",
     "function-order": "error",
```

Comment:
A lot of these others are also handled by prettier, but I don't think it's a huge deal.